backend/api/main.py

```diff
@@ -13,7 +13,7 @@
 from autogen_agentchat.base import TaskResult
 
 # Import predefined models and manager
-from backend.core.agents import AgentManager, IssueRequest, IssueResponse
```

Comment:
Wouldn't this give an error on the Openshift pod, since it would run  from the root of the repository? 

For a local run, would running the server from root of the repository instead of the backend directory solve this issue?

Reply:
It looks good overall, If we revert this change and update the README, I'll go ahead and merge this in. 
Thanks for the fixes!